distribution/openrct2.d.ts

```diff
@@ -1064,7 +1070,12 @@ declare global {
         "jumping_fountain_water" |
         "litter" |
         "money_effect" |
+        /**
+         * @deprecated since version 34, use guest or staff instead.
+         */
```

Comment:
Just to note, the `@deprecated` tag doesn't work for values. It's still good to mark them as such of course.

Reply:
Is it possible to move the `peep` entry to the bottom of the list? It would make the deprecation warning much less ambiguous.